contributing.md

```diff
@@ -1,5 +1,22 @@
 Contributing
 ===
+Welcome to the world of content development in High Fidelity! This project is actively maintained by the High Fidelity Experiences development team and we welcome contributions of content that you'd like to share with the community under the [Apache 2.0 License](https://github.com/highfidelity/hifi-content/blob/master/LICENSE), either as part of your own work or through a request made as part of the [High Fidelity Development Fund](https://blog.highfidelity.com/cashing-out-earn-and-exchange-high-fidelity-coins-hfc-for-u-s-dollars-bfa3b8cb3ebc).
+
+The High Fidelity Development Fund
+===
+The High Fidelity Development Fund is a way for members of our community can take on projects to extend the High Fidelity platform. There are dozens of features we’re planning to add, and we’re looking to developers in our community to sign up to create them. Of course, we’re planning to pay them for their hard work — in HFC, which they can convert to legal tender.
+
+We’ve set up a [public group on Telegram](https://t.me/highfidelityvr) which developers can use to contact us about these projects. We plan to regularly post new opportunities to the channel for people to select and bid for. The aim here is to reward our most dedicated members, to accelerate feature rollout, and to inject more activity into the economy, which in turn will benefit everyone.
```

Comment:
**Wordsmithing:** We ~~’ve set up~~ **have** a [public group on Telegram](https://t.me/highfidelityvr) which developers ~~can~~ use to contact us about these projects. We ~~plan to~~ regularly post new opportunities to the channel for people to select and bid for. The aim here is to reward our most dedicated members, to accelerate feature rollout, and to inject more activity into the economy, which in turn will benefit everyone.